flytekit/sdk/types.py

```diff
@@ -1,8 +1,15 @@
 from __future__ import absolute_import
 from flytekit.common.types import primitives as _primitives, blobs as _blobs, schema as _schema, helpers as _helpers, \
     proto as _proto, containers as _containers
+import enum
 
 
+class SparkType(enum.Enum):
```

Comment:
Can this be defined in flyteidl?

Reply:
Looks like it is and you map it above. This file is really for I/O typing so I wouldn't introduce them here.

Reply:
will move to a different file